hyperspy/model.py

```diff
@@ -107,6 +107,16 @@ def _twinned_parameter(parameter):
 
 
 def reconstruct_component(comp_dictionary, **init_args):
+    # Restoring of Voigt and Arctan components saved with Hyperspy <v1.6
```

Comment:
I am wondering, if this "conversion" should be logged at the `INFO` level to avoid doing things silently?